content/docs/command-reference/checkout.md

```diff
@@ -207,8 +200,29 @@ $ md5 model.pkl
 MD5 (model.pkl) = 98af33933679a75c2a51b953d3ab50aa
 ```
 
-What happened is that DVC went through the DVC-files and adjusted the current
-set of <abbr>output</abbr> files to match the `outs` in them.
+DVC went through `dvc.lock` and adjusted the current set of <abbr>outputs</abbr>
+to match the `outs` in it.
+
+## Example: Specific files or directories
+
+`dvc checkout` only affects the tracked data corresponding to any given
+`targets`:
+
+```dvc
+$ git checkout master
+$ dvc checkout            # Start with latest version of everything.
+
+$ git checkout baseline-experiment -- dvc.lock
+$ dvc checkout model.pkl  # Get previous model file only.
+```
+
+Note that granular files inside directories tracked as a whole are supported.
```

Comment:
it should be at least `checking out files granularly  ...` ... the way it's written if feels like semantics is not right ... also, let's try to avoid this? can we do plain english?

Reply:
I think "granular files inside directories" is correct. But anyway, yes: I was attached to this term from the title of this PR and original issue, but in reality we don't need it. Simplified now (in a36ba41f as well).